README.md

```diff
@@ -1,364 +1,370 @@
 ### 4.4. Quorum
 Ten percent or thirty (whichever is greater) of Full Members or General Members of the Society shall form a quorum of a General Meeting. If the quorum has not been met, then the General Meeting is failed.
 ### 4.5. Annual General Meeting
-4.5.1. The Annual General Meeting shall be convened by the Chairperson of the Society and shall be held within 3 weeks after the Election.  
-4.5.2. If there is other business raised during the Annual General Meeting, all items can only be discussed in the next Extraordinary General Meeting, together with those items listed in the agenda of the next Extraordinary General Meeting.  
-4.5.2.1. To receive and adopt the minutes of the previous Annual General Meeting together with the minutes of all Extraordinary General Meetings, which may have held since the previous annual General Meeting;  
-4.5.2.2. To receive and adopt the Annual Report of the Secretary for the current session of the Society;  
-4.5.2.3. To receive and adopt the Financial Report including an Audited Financial Statement and a Balance Sheet of the Financial Secretary of the current session of the Society;  
-4.5.2.4. To discuss to have non-confidence with an Executive Committee, if any;  
-4.5.2.5. To announce the Honorary Member(s);  
-4.5.2.6. To introduce the Executive Committee of the next session;  
-4.5.2.7. To pass and approve or amend the year plan and financial budget of the Executive Committee of the next session;  
-4.5.2.8. To pass amendments to the Constitutions, if any;  
-4.5.2.9. To appoint an Honorary Advisor and/or an Honorary Auditor;  
+The Annual General Meeting shall be convened by the President of the Society and shall be held within 3 weeks after the Election.  
+If there is other business raised during the Annual General Meeting, all items can only be discussed in the next Extraordinary General Meeting, together with following items listed in the agenda of the next Extraordinary General Meeting.
+1. To receive and adopt the minutes of the previous Annual General Meeting together with the minutes of all Extraordinary General Meetings, which may have held since the previous annual General Meeting.
+2. To receive and adopt the Annual Report of the Secretary for the current session of the Society.
+3. To receive and adopt the Financial Report, including an Audited Financial Statement and a Balance Sheet of the Financial Secretary of the current session of the Society.
+4. To discuss to have non-confidence with an Executive Committee, if any.
+5. To announce the Honorary Member(s).
+6. To introduce the Executive Committee of the next session.
+7. To pass and approve or amend the year plan and financial budget of the Executive Committee of the next session.
+8. To pass amendments to the Constitution, if any.
```

Comment:
To approve proposed changes in membership fee to become effective in the upcoming session, if any.